packages/terra-dialog-modal/src/DialogModal.scss

```diff
@@ -0,0 +1,91 @@
+:local {
+  .dialog-modal-wrapper {
+    background-color: var(--terra-dialog-modal-background-color, #fff);
+    border-radius: var(--terra-dialog-modal-border-radius, 0.3125rem);
+
+
+    &.width-320 {
+      width: 320px;
+    }
+
+    &.width-480 {
+      width: 480px;
+    }
+
+    &.width-640 {
+      width: 640px;
+    }
+
+    &.width-800 {
+      width: 800px;
+    }
+
+    &.width-960 {
+      width: 960px;
+    }
+
+    &.width-1120 {
+      width: 1120px;
+    }
+
+    &.width-1280 {
+      width: 1280px;
+    }
+
+    &.width-1440 {
+      width: 1440px;
+    }
+
+    &.width-1600 {
+      width: 1600px;
+    }
+
+    &.width-1760 {
+      width: 1760px;
+    }
+
+    &.width-1920 {
+      width: 1920px;
+    }
+
+    @media screen and (max-width: 768px) {
+      width: calc(100% - 2 * var(--terra-dialog-modal-max-width, 10px));
```

Comment:
update variable name from `--terra-dialog-modal-max-width` to `--terra-dialog-modal-small-horizontal-inset`